apps/portal/js/wizard.js

```diff
@@ -332,9 +332,21 @@ function getProviderConfig() {
         contentType: "application/json",
         async: false,
         success: function (data) {
-            registerAdvancedProviderUI(data);
             var providerHbs = Handlebars.compile($('#ui-config-hbs').html());
-            $("#provider-config").html(providerHbs(data));
+            if(Object.keys(data[0]).indexOf("grouping")>-1){
```

Comment:
Formatting issue. There should be a space between the > and -1. Add "grouping" as constant and use.